src/file_upload.rs

```diff
@@ -37,7 +37,7 @@ async fn save_file(
 /// Create new future to handle file as multipart data.
```

Comment:
While not technically part of this PR (and we can do this in a follow-up), any chance you could write a better doc comment here? `save_file()` also has a pretty poor doc comment right now. :/